### PR TITLE
Relax fun_with_flags dependency

### PR DESCRIPTION
**Pull request by tomtaylor**
This allows this library to work with the `fun_with_flags` 1.5 series and upwards.

### CODE REVIEW

**Comment by tomtaylor** [comment]
@naps62 it'd be really great to get this merged in, unless you've got any concerns?

File: mix.exs [hunk 1]
```diff
@@ -26,7 +26,7 @@ defmodule FunWithFlagsInMemory.MixProject do
     [
       {:ex_doc, "~> 0.19", only: :dev},
       {:ecto, "~> 3.0"},
-      {:fun_with_flags, "~> 1.4.0"}
+      {:fun_with_flags, "~> 1.4"}
```
Comment (naps62) [inline mix.exs]:
```suggestion
      {:fun_with_flags, ">= 1.4.0 and < 1.6.0"}
```

**Comment by naps62** [comment]
@tomtaylor thanks for the PR!
I'd be happy to merge, but I'm afraid of relaxing the version too much. Would be okay with the suggestion I proposed above? Essentially allowing all 1.4 and 1.5

The reason being that this library uses private APIs of the package, which can very easily change and break this. So allowing all versions from now on seems overly-optimistic

**Comment by naps62** [comment]
@tomtaylor I went with my suggestions, since you haven't replied yet

Version 0.1.1 is published now, so feel free to use it :)

**Pull request merged**